src/FunctionApps/python/tests/IntakePipeline/pipeline_test.py

```diff
@@ -6,48 +6,64 @@
 TEST_ENV = {
     "INTAKE_CONTAINER_URL": "some-url",
     "INTAKE_CONTAINER_PREFIX": "some-prefix",
-    "OUTPUT_CONTAINER_PATH": "output/path",
+    "VALID_OUTPUT_CONTAINER_PATH": "output/valid/path",
+    "INVALID_OUTPUT_CONTAINER_PATH": "output/invalid/path",
     "HASH_SALT": "super-secret-definitely-legit-passphrase",
     "SMARTYSTREETS_AUTH_ID": "smarty-auth-id",
     "SMARTYSTREETS_AUTH_TOKEN": "smarty-auth-token",
     "FHIR_URL": "fhir-url",
 }
 
+MESSAGE_MAPPINGS = {
+    "bundle_type": "VXU",
+    "root_template": "VXU_V04",
+    "input_data_type": "Hl7v2",
+    "template_collection": "microsofthealth/fhirconverter:default",
+    "filename": "some-filename-1",
+}
+
 
-@mock.patch("IntakePipeline.read_fhir_bundles")
 @mock.patch("IntakePipeline.transform_bundle")
 @mock.patch("IntakePipeline.add_patient_identifier")
 @mock.patch("IntakePipeline.upload_bundle_to_fhir_server")
-@mock.patch("IntakePipeline.store_bundle")
-@mock.patch("IntakePipeline.get_fhirserver_cred_manager")
+@mock.patch("IntakePipeline.store_data")
 @mock.patch("IntakePipeline.get_smartystreets_client")
+@mock.patch("IntakePipeline.convert_message_to_fhir")
 @mock.patch.dict("os.environ", TEST_ENV)
 def test_basic_pipeline(
+    patched_converter,
     patched_get_geocoder,
-    patched_get_fhirserver_cred_manager,
     patched_store,
     patched_upload,
     patched_patient_id,
     patched_transform,
-    patched_fhir_read,
 ):
+    patched_converter.return_value = {"hello": "world"}
 
     patched_geocoder = mock.Mock()
     patched_get_geocoder.return_value = patched_geocoder
 
-    patched_fhirserver_cred_manager = mock.Mock()
-    patched_get_fhirserver_cred_manager.return_value = patched_fhirserver_cred_manager
-
-    patched_fhir_read.return_value = [("VXU", {"hello": "world"})]
-    run_pipeline()
+    cred_manager = mock.Mock()
+    run_pipeline(
+        "MSH|Hello World", MESSAGE_MAPPINGS, "some-fhir-url", "some-token", cred_manager
+    )
 
     patched_get_geocoder.assert_called_with("smarty-auth-id", "smarty-auth-token")
-    patched_fhir_read.assert_called_with("some-url", "some-prefix")
+    patched_converter.assert_called_with(
+        message="MSH|Hello World",
+        input_data_type=MESSAGE_MAPPINGS["input_data_type"],
```

Comment:
`MESSAGE_MAPPINGS` isn't being used elsewhere, right? Let's inline it (no need to build out the dict, it's a test)